Make the test script's assert() function
- not print the first 50 lines for every tmp file it created so far
- continue the script to actually run further tests

Early tmux(1) failure due PORTS_PRIVSEP disabling _pbuild's shell
effectively skipped later tests.

Index: system_test.sh
--- system_test.sh.orig
+++ system_test.sh
@@ -24,6 +24,9 @@ function assert {
 	[[ "$1" == "$2" ]] && { printf "."; return; }
 	printf "\nFAIL: $this\n'$1' != '$2'\n"
 
+	# account for failure to be skipped over
+	let tests-=1
+	return
 	echo "*************************************"
 	echo "System test directory contents:"
 	head -n 50 $(find $tmp -type f)
